public/assets/css/header.css

```diff
@@ -1,40 +1,37 @@
-header{
+.topHeader{
     display: flex;
     align-items: center;
     justify-content:space-between;
-    background-color: #facfcf;
-    font-family: 'Mali';
+    background-color: var(--third-color);
+    font-family: var(--primary-font);
     font-size: 25px;
     font-weight: bold;
   }
 
-  aside{
-    background-color: #facfcf;
-  }
-  
-  header a img{
+  .topHeader a img{
     width: 70px;
     height: 70px;
   }
   
   a{
```

Comment:
If all the a tag have to take this behavior could you put it inside style.css which is the file for tags and class global behavior on all website